index.md

```diff
@@ -193,10 +193,17 @@ Simon Brand, Fraser Cormack, Sean Jones, Mark M. Miller, Meenakshi Ravindran, Ta
 -----------------
 
 ## Glossary
+
 Brexit - Voting down three governments in a row, thus causing a policy to be revealed at random. So-called because some players would vote “no” in order to register dissent, assuming that everyone else would vote yes, but if too many people do this, then the government will actually fail.
 
 Breakfast - Another name for Brexit
 
 Chain investigation - The second investigator in a 9/10 player game investigating the first investigator.
 
 Cut out - Someone who is still alive, but is no longer considered for Governments by the table due to them being in a dispute.
+
+Liberal block/corner - A perceived group of Liberal players who are trusted by the majority
+
+Maverick cop - Someone who throws out crazy theories which might just be true
```

Comment:
Should it not be, someone who throws out unsubstantiated claims and their belief cannot be shaken?